README.md

```diff
@@ -145,3 +145,14 @@ $ npm publish
 A [live version](https://checkout-demo.easypay.pt/) of an integration with the SDK is available.
 
 To test the package locally, you can use the source code for the [checkout-demo](https://github.com/Easypay/checkout-demo) as a starting point.
+
+## Common Problems
+
+There are a few problems that may come up while integrating the checkout with your app.
```

Comment:
I would just capitalize **C**heckout and remove `the` (as in: `integrating Checkout`). The same is true for the rest of the text below.